File: rails-admin/app/models/report.rb
# coding: utf-8
class Report < ActiveRecord::Base
  nilify_blanks
  self.table_name = 'report'
  belongs_to :conference
  belongs_to :user

  rails_admin do
    label "Доклад"
    label_plural "Доклады"
    object_label_method :title

    list do
      field :id
      field :title do
        label "Название"
      end
      field :conference do
        label "конференция"
      end
    end

    edit do
      field :title do
        label "Название"
      end
      field :conference do
        label "конференция"
      end
      field :description, :text do
        label "описание"
      end
      field :presentation do
        label "презентация"
      end
      field :video do
        label "видео"
      end
      field :other_conferences
      field :key_technologies
      field :reporter_wishes
    end
  end
end

File: src/main/java/it/sevenbits/conferences/model/Report.java
package it.sevenbits.conferences.model;


/**
 * Reports Model.
 */
public class Report {

    private Long id;
    private Conference conference;
    private User user;
    private String title;
    private String description;
    private String presentation;
    private String video;
    private String otherConferences;
    private String keyTechnologies;
    private String reporterWishes;


    public Long getId() {
        return id;
    }

    public void setId(final Long id) {
        this.id = id;
    }


    public Conference getConference() {
        return conference;
    }

    public void setConference(final Conference conference) {
        this.conference = conference;
    }


    public User getUser() {
        return user;
    }

    public void setUser(final User user) {
        this.user = user;
    }


    public String getTitle() {
        return title;
    }

    public void setTitle(final String title) {
        this.title = title;
    }


    public String getDescription() {
        return description;
    }

    public void setDescription(final String description) {
        this.description = description;
    }


    public String getPresentation() {
        return presentation;
    }

    public void setPresentation(final String presentation) {
        this.presentation = presentation;
    }


    public String getVideo() {
        return video;
    }

    public void setVideo(final String video) {
        this.video = video;
    }


    public String getOtherConferences() {
        return otherConferences;
    }

    public void setOtherConferences(final String otherConferences) {
        this.otherConferences = otherConferences;
    }


    public String getKeyTechnologies() {
        return keyTechnologies;
    }

    public void setKeyTechnologies(final String keyTechnologies) {
        this.keyTechnologies = keyTechnologies;
    }


    public String getReporterWishes() {
        return reporterWishes;
    }

    public void setReporterWishes(final String reporterWishes) {
        this.reporterWishes = reporterWishes;
    }
}
